### DISCUSSION TITLE
theme override.

### DISCUSSION THREAD

**Discussion by semi710** [nix-community/stylix#455] [Q&A]
I have searched discussion and issues and docs, didn't find any relevant answers.
Maybe it is on `base16.nix` but then also creating issue to ask it here.

![image](https://github.com/danth/stylix/assets/60490474/ce3e25c5-4b26-49fe-8791-7cf373dae97d)
here is how my discord is looking, I want to have the same color for the tab where channels are listed.

same for nautilus
![image](https://github.com/danth/stylix/assets/60490474/fee7ddf3-b0b7-4656-9c02-ccf86c405561)

is there any way by which i can override this behavior? or theme?

**Comment by danth** [comment]
The background used for the sidebar is `base01`. The following setting will make everything use the same background color:

```nix
{ config, ... }:

{
  stylix.override.base01 = config.lib.stylix.colors.base00;
}
```

It'll affect everything, so some other apps may look less than ideal.

**Reply by semi710** [reply]
There ain't any way to do this modification for specific target?

e.g `stylix.target.override.base01 = config.lib.stylix.colors.base00;`

**Reply by danth** [reply]
Not currently; see #208

**Reply by semi710** [reply]
@danth the `  stylix.override.base01 = config.lib.stylix.colors.base00;` returns in infinite recursion.

**Comment by GideonWolfe** [comment]
@niksingh710 what Icon theme are you using? I'm looking for one that takes Stylix colors

**Reply by semi710** [reply]
no icon theme resutls in that.
